gyp info it worked if it ends with ok
gyp info using node-gyp@11.0.0
gyp info using node@22.14.0 | linux | x64
gyp info find Python using Python version 3.7.6 found at "/home/xarkqngel/anaconda3/bin/python3"
gyp http GET https://nodejs.org/download/release/v22.14.0/node-v22.14.0-headers.tar.gz
gyp http 200 https://nodejs.org/download/release/v22.14.0/node-v22.14.0-headers.tar.gz
gyp http GET https://nodejs.org/download/release/v22.14.0/SHASUMS256.txt
gyp http 200 https://nodejs.org/download/release/v22.14.0/SHASUMS256.txt
gyp info spawn /home/xarkqngel/anaconda3/bin/python3
gyp info spawn args [
gyp info spawn args '/home/xarkqngel/.local/share/mise/installs/node/22.14.0/lib/node_modules/npm/node_modules/node-gyp/gyp/gyp_main.py',
gyp info spawn args 'binding.gyp',
gyp info spawn args '-f',
gyp info spawn args 'make',
gyp info spawn args '-I',
gyp info spawn args '/home/xarkqngel/Projects/xark-mapbox-backend/node_modules/socket.io-client/node_modules/ws/build/config.gypi',
gyp info spawn args '-I',
gyp info spawn args '/home/xarkqngel/.local/share/mise/installs/node/22.14.0/lib/node_modules/npm/node_modules/node-gyp/addon.gypi',
gyp info spawn args '-I',
gyp info spawn args '/home/xarkqngel/.cache/node-gyp/22.14.0/include/node/common.gypi',
gyp info spawn args '-Dlibrary=shared_library',
gyp info spawn args '-Dvisibility=default',
gyp info spawn args '-Dnode_root_dir=/home/xarkqngel/.cache/node-gyp/22.14.0',
gyp info spawn args '-Dnode_gyp_dir=/home/xarkqngel/.local/share/mise/installs/node/22.14.0/lib/node_modules/npm/node_modules/node-gyp',
gyp info spawn args '-Dnode_lib_file=/home/xarkqngel/.cache/node-gyp/22.14.0/<(target_arch)/node.lib',
gyp info spawn args '-Dmodule_root_dir=/home/xarkqngel/Projects/xark-mapbox-backend/node_modules/socket.io-client/node_modules/ws',
gyp info spawn args '-Dnode_engine=v8',
gyp info spawn args '--depth=.',
gyp info spawn args '--no-parallel',
gyp info spawn args '--generator-output',
gyp info spawn args 'build',
gyp info spawn args '-Goutput_dir=.'
gyp info spawn args ]
Traceback (most recent call last):
  File "/home/xarkqngel/.local/share/mise/installs/node/22.14.0/lib/node_modules/npm/node_modules/node-gyp/gyp/gyp_main.py", line 42, in <module>
    import gyp  # noqa: E402
  File "/home/xarkqngel/.local/share/mise/installs/node/22.14.0/lib/node_modules/npm/node_modules/node-gyp/gyp/pylib/gyp/__init__.py", line 9, in <module>
    import gyp.input
  File "/home/xarkqngel/.local/share/mise/installs/node/22.14.0/lib/node_modules/npm/node_modules/node-gyp/gyp/pylib/gyp/input.py", line 8, in <module>
    import gyp.common
  File "/home/xarkqngel/.local/share/mise/installs/node/22.14.0/lib/node_modules/npm/node_modules/node-gyp/gyp/pylib/gyp/common.py", line 435
    if CC := os.environ.get("CC_target") or os.environ.get("CC"):
           ^
SyntaxError: invalid syntax
gyp ERR! configure error 
gyp ERR! stack Error: `gyp` failed with exit code: 1
gyp ERR! stack at ChildProcess.<anonymous> (/home/xarkqngel/.local/share/mise/installs/node/22.14.0/lib/node_modules/npm/node_modules/node-gyp/lib/configure.js:317:18)
gyp ERR! stack at ChildProcess.emit (node:events:518:28)
gyp ERR! stack at ChildProcess._handle.onexit (node:internal/child_process:293:12)
gyp ERR! System Linux 6.11.0-21-generic
gyp ERR! command "/home/xarkqngel/.local/share/mise/installs/node/22.14.0/bin/node" "/home/xarkqngel/.local/share/mise/installs/node/22.14.0/lib/node_modules/npm/node_modules/node-gyp/bin/node-gyp.js" "rebuild"
gyp ERR! cwd /home/xarkqngel/Projects/xark-mapbox-backend/node_modules/socket.io-client/node_modules/ws
gyp ERR! node -v v22.14.0
gyp ERR! node-gyp -v v11.0.0
gyp ERR! not ok 
